gallery/visualtests/testUtils.js

```diff
@@ -75,13 +75,13 @@ module.exports = {
       await page.mouse.move(0, 0);
     });
 
-    async function overrideDisplayTheme(theme) {
+    async function setThemeOverride(theme) {
       await page.evaluate((t) => {
         window.setThemeOverride(t);
       }, theme);
     }
 
-    async function removeThemeEnabling() {
+    async function setThemingEnabled() {
```

Comment:
This needed to not just be renamed but also to take the boolean parameter

Reply:
> This needed to not just be renamed but also to take the boolean parameter

Sorry, you beat me to the push of the updated function: [5740744](https://github.com/sonatype/sonatype-react-shared-components/pull/808/commits/574074445b615d7b483810cd4cb99bfa87a28506)